ClientApp/src/api-authorization/logout/logout.component.ts

```diff
@@ -94,8 +95,8 @@ export class LogoutComponent implements OnInit {
     });
   }
 
-  private getReturnUrl(state?: INavigationState): string {
-    const fromQuery = (this.activatedRoute.snapshot.queryParams as INavigationState).returnUrl;
+  private getReturnUrl(state?: NavigationState): string {
```

Comment:
Codacy found an issue: [Method 'getReturnUrl' has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6277153)